segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -219,7 +220,7 @@ public void readSegment(ReadSegment readSegment) {
                                                          wrapCancellationException(ex)));
     }
 
-    private boolean verifyToken(String segment, long requestId, String delegationToken, String operation) {
+    protected boolean verifyToken(String segment, long requestId, String delegationToken, String operation) {
```

Comment:
Why did you make all of these protected/public?

Reply:
`AdminRequestProcessor` extends `PravegaRequestProcessor`. I've changed these to `protected` in order to be able to use them in `AdminRequestProcessor`.